### PR TITLE
Add an RLlib Tune experiment to UserTest suite.

### PR DESCRIPTION
**Pull request by gjoliver**
Add RLlib & Tune UserTest.

## Why are these changes needed?

To make sure client connect works.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [*] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/rllib_tests/4gpus_64cpus.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
 region: us-west-2
 
-max_workers: 0
+max_workers: 5
```
Comment (amogkam) [inline release/rllib_tests/4gpus_64cpus.yaml:4]: Are these changes intentional? It seems like only auto_scale.yaml is needed for the test.

**Review by amogkam — APPROVED** [review]
LGTM! Just want to make sure if the changes to the non auto-scale.yaml compute templates are intentional.

**Comment by gjoliver** [comment]
Thanks Amog, for adding the driver requirement file. Good to know.

I just gave this a try, and kicking off a manual run from Buildkite fails with
No such file or directory: 'release/rllib_tests/connect_driver_requirements.txt'

Looking at the command line, it seems maybe we have to merge connect_driver_requirements.txt first before we can test this?
It seems to pip install requirement files before cloning the PR branch.

Thanks again.

File: release/rllib_tests/4gpus_64cpus.yaml (hunk 1 above)
Comment (gjoliver) [reply release/rllib_tests/4gpus_64cpus.yaml:4]: ah yes. I switched around. I actually used all of these before settling on a new autoscale.yaml config.
these configs don't work without the changes. max_workers==0 and no attached disk, etc.
they are not used right now. so I am fine with keeping them as is, or simply deleting them, whichever you prefer.

**Comment by amogkam** [comment]
Trying one more time here: https://buildkite.com/ray-project/periodic-ci/builds/1470#3a68464a-a406-4ba4-8f9e-8fb0fce8be47
